pkg/containerutil/container_network_manager.go

```diff
@@ -150,6 +150,11 @@ type noneNetworkManager struct {
 	client        *containerd.Client
 }
 
+func WriteDefaultResolvConf(filePath string) error {
```

Comment:
I think we should use the existing `resolvconf.Build`.
It is not currently safe to use it (not atomic, nor locking), but it is better IMO than duplicating the problem.